### ISSUE TITLE
Occasional NPE stopping builds from succeeding

### ISSUE THREAD

**Issue by grandstaish** [TestArmada/flank#99]
Version: 1.7.0

```
java.lang.NullPointerException
	at com.walmart.otto.reporter.PriceReporter.getBillableTime(PriceReporter.java:57)
	at com.walmart.otto.reporter.PriceReporter.getTotalBillableTime(PriceReporter.java:40)
	at com.walmart.otto.Flank.printEstimates(Flank.java:148)
	at com.walmart.otto.Flank.start(Flank.java:83)
	at com.walmart.otto.Flank.main(Flank.java:107)
```

Could we make it so the build doesn't fail and the billable time was just left unknown?

**Comment by runningcode** [comment]
But then the bug will never be fixed! :O

**Comment by renas** [comment]
Thanks for reporting this. We will fix this shortly and release 2.0 after its completed.

**Comment by renas** [comment]
Fixed in 2.0.0
